rollup.config.js

```diff
@@ -0,0 +1,74 @@
+import { defineConfig } from 'rollup'
+import nodeResolve from '@rollup/plugin-node-resolve'
+import babel from '@rollup/plugin-babel'
+import replace from '@rollup/plugin-replace'
+import typescript from 'rollup-plugin-typescript2'
+import { terser } from 'rollup-plugin-terser'
+
+const extensions = [ '.ts' ],
+	overrideTSConfig = { compilerOptions: { declaration: true }, declarationMap: false },
```

Comment:
```suggestion
	overrideTSConfig = { compilerOptions: { declaration: true }, declarationMap: true },
```
Try to publish the package, and see if `declarationMap` give any better results.